packages/ui-tests/cypress/e2e/designer/catalogs/springBootCatalog.cy.ts

```diff
@@ -0,0 +1,50 @@
+describe('Tests for SpringBoot catalog type', () => {
+  beforeEach(() => {
+    cy.openHomePage();
+  });
+
+  const runtime = 'SpringBoot';
+
+  it('Camel SpringBoot catalog type with CR', () => {
+    cy.selectRuntimeVersion(runtime);
+    cy.uploadFixture('flows/camelRoute/basic.yaml');
+    cy.openDesignPage();
+
+    cy.hoverOnRuntime(runtime);
+    cy.get(`[data-testid^="runtime-selector-Camel ${runtime}"]`).then(($element) => {
+      const dataTestidValue = $element.attr('data-testid');
+      const elementVersion = dataTestidValue!.substring(dataTestidValue!.lastIndexOf(' ') + 1);
+      cy.selectAppendNode('setHeader');
+      cy.checkCatalogVersion(elementVersion);
+    });
+
+    cy.chooseFromCatalog('component', 'as2');
```

Comment:
What do you think about testing with a component which is available only in SpringBoot catalog? It would ensure that the correct catalog is loaded.
I'm not sure that there is one unfortunately. But maybe a component not available in Quarkus, such as spring-event

Reply:
this is a good point - changed to `spring-event`